app/models/krikri/activity.rb

```diff
@@ -83,5 +83,23 @@ def rdf_subject
       RDF::URI(Krikri::Settings['marmotta']['ldp']) /
         Krikri::Settings['prov']['activity'] / id.to_s
     end
+
+    ##
+    # Return an Enumerator of URI strings of entities (e.g. aggregations or
+    # original records) that pertain to this activity
+    #
+    # @return [Enumerator] URI strings
+    def generated_entity_uris
```

Comment:
I think renaming this would be best, because "generated" could definitely be confusing for someone new to the code, if there are expectations set elsewhere for a distinction between "revised" and "generated."  There aren't too many lines that need changing.